src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -181,13 +181,10 @@ def put(
 
                 if result.get("regenerate"):
                     if request.access.has_scope("member:admin"):
-                        region_outbox = None
                         with transaction.atomic():
                             member.regenerate_token()
                             member.save()
-                            region_outbox = member.save_outbox_for_update()
-                        if region_outbox:
-                            region_outbox.drain_shard(max_updates_to_drain=10)
+                        member.outbox_for_update().drain_shard(max_updates_to_drain=10)
```

Comment:
```suggestion 
                        # Use outbox_for_update() to build an unsaved outbox message that we can use to drain with
                        member.outbox_for_update().drain_shard(max_updates_to_drain=10)
```

This was confusing on first read, as it looks like we're missing a save() call on the outbox message. I realized afterwards that you're using `outbox_for_update()` to rebuild the attributes necessary to drain the messages, perhaps we could include a comment for future us?